### DISCUSSION TITLE
Interior boundaries

### DISCUSSION THREAD

**Discussion by griff10000** [inducer/meshpy#83] [Q&A] [answered]
Does `meshpy` have a way of identifying interior boundary nodes?

For, say, the given example of a rectangle with two circular holes.

**Comment by inducer** [comment]
https://github.com/inducer/meshpy/blob/19588442f5f400126b6529c5cc03907ba842215d/examples/test_triangle.py#L29

**Reply by griff10000** [reply]
Thank you for your reply.
I wonder if you could expand your answer to provide an example of how I can extract node indices for hole boundaries.

I have been using examples from` jw_meshtools.py`

**Reply by inducer** [reply]
Look at facet attributes.

**Reply by griff10000** [reply, marked as answer]
Thanks. I have now found a solution using jw_meshtools.py
